content/en/tools/cloud-pods/pods-cli/index.md

```diff
@@ -80,17 +80,20 @@ Options:
 The `inject` command let users inject a specific application state, previously saved, into the application runtime.
 Please note that this is a local-only operation, i.e., the injecting state must be located on the host machine (usually under `~/.localstack/cloudpods/<pod_name>`).
 
-By default, the injecting state will replace the application runtime.
-The `--merge` option, instead, will first merge the injecting state with the current runtime and then inject the result.
+By default, the injecting state updates the application runtime at a service level.
+Formally known as the `merge` injection strategy, the state of all services specified in the injecting state are reflected exactly in the application runtime, whilst all other active services remain unchanged.
```

Comment:
nit: we can avoid comparisons with the previous usage to avoid confusion for the users

Reply:
Yes I agree! I wanted to communicate how the option is formally named 'merge'; hope now it's less confusing